docs/source/notebooks/ModelBuilder in PyMC-Marketing context.ipynb

```diff
@@ -0,0 +1,1149 @@
+{
```

Comment:
<p>I think this intro paragraph can be made shorter and stronger. No need to be shy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>how do you like the new hopefully-not-so-shy version? :)</p>